tests/csapi/rooms_members_local_test.go

```diff
@@ -4,20 +4,55 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/tidwall/gjson"
+
 	"github.com/matrix-org/complement/internal/b"
 	"github.com/matrix-org/complement/internal/client"
-	"github.com/matrix-org/complement/runtime"
-	"github.com/tidwall/gjson"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
 )
 
 func TestMembersLocal(t *testing.T) {
-	deployment := Deploy(t, b.BlueprintOneToOneRoom)
+	deployment := Deploy(t, b.MustValidate(b.Blueprint{
+		Name: "ab",
+		Homeservers: []b.Homeserver{
+			{
+				Name: "hs1",
+				Users: []b.User{
+					{
+						Localpart:   "@alice",
+						DisplayName: "Alice",
+					},
+					{
+						Localpart:   "@bob",
+						DisplayName: "Bob",
+					},
+				},
+			},
+		},
+	}))
 	defer deployment.Destroy(t)
 
 	alice := deployment.Client(t, "hs1", "@alice:hs1")
 	bob := deployment.Client(t, "hs1", "@bob:hs1")
 	roomID := alice.CreateRoom(t, map[string]interface{}{"preset": "public_chat"})
 
+	res := bob.DoFunc(
+		t,
+		"PUT",
+		[]string{"_matrix", "client", "v3", "presence", bob.UserID, "status"},
+		client.WithJSONBody(t, map[string]interface{}{
+			"presence": "online",
+		}),
+	)
+	must.MatchResponse(t, res, match.HTTPResponse{
+		StatusCode: 200,
+	})
```

Comment:
This can be replaced by a simple `bob.MustSync(t, client.SyncReq{})` (optionally set the presence there)

Reply:
The original sytest explicitly uses a presence-setting url, but I can change it to that.